packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.gen.ts

```diff
@@ -503,11 +503,9 @@ export const defaultVizConfigKind = (): VizConfigKind => ({
 });
 
 export interface AnnotationQuerySpec {
-	datasource: DataSourceRef;
+	datasource?: DataSourceRef;
 	query: DataQueryKind;
-	// TODO: Should be figured out based on datasource (Grafana ds)
-	// builtIn?: int
-	// Below are currently existing options for annotation queries
+	builtIn: boolean;
```

Comment:
Let's make it optional as it's only going to exist on one annotation query really.